<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPresetDialog</class>
 <widget class="QDialog" name="AddPresetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>273</width>
    <height>111</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddPresetDialog</string>
  </property>
  <widget class="QLabel" name="id_lab_preset">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>预置点别名:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="id_btn_sure">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="id_btn_cancel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="id_edit_preset">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>110</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
